src/styles/base/_base.scss

```diff
@@ -0,0 +1,27 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 10px;
+
+  @extend %font-roboto;
+}
+
+.button {
+  display: block;
+  width: 100%;
+  border: none;
+  cursor: pointer;
+}
+
+img {
+  width: 100%;
+  height: 100%;
```

Comment:
Setting both `width` and `height` to `100%` for all `img` elements can cause distortion if the aspect ratio of the image is not 1:1. Consider using `max-width` and `height: auto` or vice versa to maintain the image's aspect ratio.